packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -66,6 +66,20 @@ export interface StageProps {
    * You can always add more Actions later by calling {@link IStage#addAction}.
    */
   readonly actions?: IAction[];
+
+  /**
+   * Whether to disable transition to this stage
+   *
+   * @default false
+   */
+  readonly disableTransition?: boolean;
+
+  /**
+   * The reason for disabling transition to this stage
+   *
+   * @default 'Transitions disabled'
```

Comment:
I think it should be "Transition disabled" (singular), right?

Reply:
It is singular in the code, forgot to update the doc string.